spec/proceduralSpec.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace procedural\spec;
+
+use function prodecural\echoScript;
+
+describe('procedural', function () {
+  describe('->echoScript()', function () {
+    it('echos input string inside a script tag', function () {
+      $closure = function() { echoScript('baz'); };
+      expect($closure)->toEcho('<script>baz</script>');
+    });
+  });
```

Comment:
add 1 enter before new test code, so, instead of  : 

```php
describe('echoScript()', function() {
    it('echos input string inside a script tag', function() {
        skipIf(PHP_VERSION_ID < 50600);
        // do test sample here
    });
});
```

you can do:


```php
describe('echoScript()', function() {

    it('echos input string inside a script tag', function() {

        skipIf(PHP_VERSION_ID < 50600);
        // do test sample here
    });

});
```